web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -49,30 +33,19 @@ export default {
       type: Object,
       required: true
     },
-    show: {
-      type: Boolean,
-      required: false,
-      default: true
-    },
   },
   data: () => ({
     store: diffTreeStore(),
     collapsed: false,
   }),
   methods: {
-    handleClick(itemIsFile) {
-      if (itemIsFile) {
-        return;
-      }
-      this.collapsed = !this.collapsed;
-    },
     getIconForDiffType(pType) {
       const diffTypes = {
-        1: 'octicon-diff-added',
-        2: 'octicon-diff-modified',
-        3: 'octicon-diff-removed',
-        4: 'octicon-diff-renamed',
-        5: 'octicon-diff-modified', // there is no octicon for copied, so modified should be ok
+        1: {name: 'octicon-diff-added', classes: ['text', 'green']},
+        2: {name: 'octicon-diff-modified', classes: ['text', 'yellow']},
+        3: {name: 'octicon-diff-removed', classes: ['text', 'red']},
+        4: {name: 'octicon-diff-renamed', classes: ['text', 'teal']},
+        5: {name: 'octicon-diff-modified', classes: ['text', 'green']}, // there is no octicon for copied, so modified should be ok
```

Comment:
I think that is semantically more a `rename` than a `modified`.
```suggestion
        5: {name: 'octicon-diff-renamed', classes: ['text', 'green']}, // there is no octicon for copied, so renamed should be ok
```

Reply:
I do not think "renamed" is a proper icon for "copied"

Reply:
From a semantic viewpoint, I'd argue that a `copy` action is the same as `renaming` the existing file while keeping the original file.
This can also be seen in the icon: The icon is a circled `→` for `rename`, while it is a non-descriptive circled `·` for `modified`.
I still think that rename is more fitting in this case.

Reply:
There is a "copy" svg if you like: ./public/assets/img/svg/octicon-copy.svg

Reply:
Yes, but we can't really use it here:
Whenever we use it, users expect to copy whatever precedes it by clicking on it.
That is not the case here.